backend/auth/Auth.ts

```diff
@@ -60,3 +81,15 @@ passport.deserializeUser<User>(function (user, done) {
 interface User {
   id: string;
 }
+
+// Created manual promise because util.promisify did not work with req.login
+const loginPromise = (user, request) => {
```

Comment:
What code were you trying with util.promisify? Here's what I would have tried

```js
const login = util.promisify(request.login)
await login(user)
```

Reply:
That is exactly what I tried.  Got the error: 

```Error logging in:  Error: passport.initialize() middleware not in use```

Reply:
Hmm that error is called with `req._passport` is not defined. I think I know why - it's related to the default javascript (`this`) of `request.login()` vs `login()`. In javascript, the default value for `this` changes depending on what is to the left of the period of code calling the function. So in `request.login()` the `this` is `request`, but in `login()` there is no period or anything to the left of it, so it will use the global variable as the default `this`.

To fix, bind the promisified version to the request object, which forces the `this` to always be the `request` object:

```js
const login = util.promisify(request.login).bind(request);
await login(user)
```

Reply:
This is great! I will try this fix.  Thank you.